pelican/graphql/base_gql.py

```diff
@@ -19,8 +22,11 @@ def _execute(self, query):
         var = json.loads(var)
         query["variables"] = var
```

Comment:
https://github.com/uc-cdis/guppy/blob/666b302eb21b3e3af822af79ec2c465f07443388/src/server/download.js#L10

The fallback endpoint doesn't use this variable property so I just bypass this.